lib/generators/solidus_subscriptions/install/install_generator.rb

```diff
@@ -15,6 +16,30 @@ def add_javascripts
         append_file 'vendor/assets/javascripts/spree/backend/all.js', "//= require spree/backend/solidus_subscriptions\n"
       end
 
+      def copy_starter_frontend_files
+        return if options[:frontend] != 'starter'
```

Comment:
imho `return unless options[:frontend] == 'starter'` is better for readability

Reply:
Thanks for the suggestions @Naokimi. I tend to use unless only when the condition doesn't contain a negation, that's because it would be a double negation, IMO, always harder to read. I think I took this reasoning line from this blog post years ago: https://signalvnoise.com/posts/2699-making-sense-with-rubys-unless. BTW, I'm open to invert it, let me know your thoughts.

Reply:
yeah, I'm also of the school that you should avoid negations as much as possible, so since return if options[:frontend] != 'starter'` contains a negation (NOT equal to) I'm suggesting removing the negation (-> equal to) while using `unless`

but in any case in this specific case what you wrote is still readable enough, so it wouldn't be a big deal to leave it as it is

Reply:
Oh, you are right. Personally, I still prefer the negation to be on the condition, so I think I'll leave it as is for now. But appreciate the conversation around this topic. Thanks again!